tests/components/test_logbook.py

```diff
@@ -9,25 +9,63 @@
 from datetime import timedelta
 
 import homeassistant.core as ha
+from homeassistant.components.logbook import Entry
 from homeassistant.const import (
     EVENT_STATE_CHANGED, EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
 import homeassistant.util.dt as dt_util
 from homeassistant.components import logbook
 
-from tests.common import get_test_home_assistant, mock_http_component
+from tests.common import mock_http_component
 
 
 class TestComponentHistory(unittest.TestCase):
     """ Tests homeassistant.components.history module. """
 
-    def test_setup(self):
+    def setUp(self):
         """ Test setup method. """
-        try:
-            hass = get_test_home_assistant()
-            mock_http_component(hass)
-            self.assertTrue(logbook.setup(hass, {}))
-        finally:
-            hass.stop()
+        self.hass = ha.HomeAssistant()
+        mock_http_component(self.hass)
+        logbook.setup(self.hass, {})
```

Comment:
Please make sure you wrap this in `self.assertTrue(…)` to ensure setup was correctly.